compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -141,6 +141,8 @@ public class SymbolTable {
     public final BType readonlyType = new BReadonlyType(TypeTags.READONLY, null);
     public final BType pathParamAllowedType = BUnionType.create(null,
             intType, stringType, floatType, booleanType, decimalType);
+    public final BType interpolationAllowedType = BUnionType.create(null, intType, floatType, decimalType,
```

Comment:
```suggestion
    public final BType unionsOfInterpolationAllowedType = BUnionType.create(null, intType, floatType, decimalType,
```

Reply:
Lets stick with the convention in the file. Eventhough it is a union we dont need to explicitly mention it as a part of the variable name